google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -479,6 +485,21 @@ public Builder setAckExpirationPadding(Duration ackExpirationPadding) {
       return this;
     }
 
+    /**
+     * Set the maximum period a message ack deadline will be extended.
+     *
+     * <p>It is recommended to set this value to a reasonable upper bound of the subscriber time to
+     * process any message. This maximum period avoids messages to be <i>locked</i> by a subscriber
+     * in cases when the {@link AckReply} is lost.
+     *
+     * <p>A zero duration effectively disables auto deadline extensions.
```

Comment:
I'm sorry I just realized this. We said "zero duration effectively disables auto deadline extensions" but we then check that the time is non-zero. I realize it's still possible to set the duration is something really small instead, but I think we should be consistent.